src/Raven.Server/Web/System/TcpConnectionInfoHandlerForDatabase.cs

```diff
@@ -11,12 +11,27 @@ public class TcpConnectionInfoHandlerForDatabase : DatabaseRequestHandler
         [RavenAction("/databases/*/info/tcp", "GET", AuthorizationStatus.ValidUser, EndpointType.Read, DisableOnCpuCreditsExhaustion = true)]
         public async Task Get()
         {
+            var senderUrl = HttpContext.Request.GetClientRequestedNodeUrl();
```

Comment:
IIRC this isn't the sender url, but currents node url

Reply:
done